distribution/docker/src/docker/bin/docker-entrypoint.sh

```diff
@@ -56,7 +56,8 @@ if [[ -f bin/elasticsearch-users ]]; then
   # enabled, but we have no way of knowing which node we are yet. We'll just
   # honor the variable if it's present.
   if [[ -n "$ELASTIC_PASSWORD" ]]; then
-    [[ -f /usr/share/elasticsearch/config/elasticsearch.keystore ]] || (run_as_other_user_if_needed elasticsearch-keystore create)
+    ES_PATH_CONF=${ES_PATH_CONF:-/usr/share/elasticsearch/config}
+    [[ -f $ES_PATH_CONF/elasticsearch.keystore ]] || (run_as_other_user_if_needed elasticsearch-keystore create)
```

Comment:
I believe that all you need to do to fix the merge conflict is remove `run_as_other_user_if_needed `.